src/ng/q.js

```diff
@@ -636,6 +636,34 @@ function qFactory(nextTick, exceptionHandler, errorOnUnhandledRejections) {
     return deferred.promise;
   }
 
+  /**
+   * @ngdoc method
+   * @name $q#race
+   * @kind function
+   *
+   * @description
+   * Returns a promise that is resolved similarly as the first promise to resolve.
+   *
+   * @param {Array.<Promise>|Object.<Promise>} promises An array or hash of promises.
+   * @returns {Promise} Returns a promise that will be resolved as soon as the first promise in
```

Comment:
How about something like this:

```
Returns a promise that will be settled as soon as the first promise in `promises` is settled. The value or reason will be the same as that of the first settled promise.
```

Note that the term `settled` is also used in the [spec](http://www.ecma-international.org/ecma-262/6.0/#sec-promise-objects).